WPFHospitalEditor/SearchType.cs

```diff
@@ -6,6 +6,7 @@ public enum SearchType
         EquipmentSearch,
         MedicationSearch,
         AppointmentSearch,
-        SpecialistAppointmentSearch
+        SpecialistAppointmentSearch,
+        EquipmentRelocationSearch
     }
```

Comment:
I see that SpecialistAppointmentSearch is never used, so it can maybe be deleted because it's almost same as normal appointment search.